sympy/geometry/point.py

```diff
@@ -106,7 +106,7 @@ def __new__(cls, *args, **kwargs):
     is_Point = True
 
     def __contains__(self, item):
-        return item == self
+        return item in self.args
```

Comment:
So that a point acts more like a list, since it's enumerable.  This way

    for x in Point(1,2,3):
        print(x in Point(1,2,3))

will print `True`.  It doesn't seem particularly relevant to this pull request though :-/